Sources/ShortcutFoundation/Extensions/Int+Extension.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Int+Extension.swift
+//  ShortcutFoundation
+//
+//  Created by Andreas Lif on 2022-01-12.
+//  Copyright © 2021 Shortcut Scandinavia Apps AB. All rights reserved.
+//
+
+import Foundation
+
+public extension Int {
+    
+    var isEven: Bool {
+        guard self != 0 else { return true }
```

Comment:
I don't see why we need a `guard` in here at all. In general it's good to minimize branching where possible, and I don't think that we really gain anything by avoiding a `%` operation in the `0` case, when at the same time we are adding a branch to every case. I have the same concern about the `isOdd` method.

Reply:
@jnutting Now that you mention it, I don't see the point either. I remember it not handling 0 when I made this, but it seems to work fine now, it so I guess I made a mistake somewhere. Oh well! I changed to @gabrielPeart's suggestion in 5fcd885